### PR TITLE
Config update for R5

### PR DESCRIPTION
**Pull request by yazzaoui**
@cmjc @raj-shekhar1 Please review

### CODE REVIEW

**Review by lorenzo-dev1 — APPROVED** [review]
LGTM in terms of code changes, but I did not check if the validator cryptographic data is correct.

**Review by lorenzo-dev1 — CHANGES_REQUESTED** [review]
@yazzaoui actually we need also to uncomment the piece of code about bls key unmarshalling in params/protocol_contracts.go Validate() function.

**Review by cmjc — APPROVED** [review]
LGTM. @lorenzo-dev1 - @spenot09 and @raj-shekhar1 verified the provided validator from the Community GV's when it was provided 👍

File: params/config.go [hunk 1]
```diff
@@ -62,13 +62,12 @@ var CheckpointOracles = map[common.Hash]*CheckpointOracleConfig{
...
+	NtnPrecision = big.NewInt(1_000_000_000_000_000_000)
+	Ntn1         = new(big.Int).Mul(big.NewInt(1), NtnPrecision)
+	Ntn10000     = new(big.Int).Mul(big.NewInt(10_000), NtnPrecision)
 
-	// PiccaddillyChainConfig todo: ask Raj to generate validator key for validators in the PiccaddillyChainConfig
-	// PiccaddillyChainConfig contains the chain parameters to run a node on the Piccaddilly test network.
-	PiccaddillyChainConfig = &ChainConfig{
+	// PiccadillyChainConfig contains the chain parameters to run a node on the Piccaddilly test network.
+	PiccadillyChainConfig = &ChainConfig{
 		ChainID:                 big.NewInt(65_100_001),
```
Comment (cmjc) [inline params/config.go:71]:
```suggestion
		ChainID:                 big.NewInt(65_100_0012),
```

Per APR25, the chain and network ID is `65100002`

**Review by cmjc — CHANGES_REQUESTED** [review]
The chain and network Id is 65100002 - see https://github.com/autonity/project-execution/edit/main/APRs/APR-25.md

File: params/config.go (hunk 1 above)
Comment (cmjc) [reply params/config.go:71]: @yazzaoui - note also per https://github.com/ethereum-lists/chains/pull/4346 for Bakerloo Testnet it is changed to:

```
"chainId": 65010002,
"networkId": 65010002,
```

**Comment by raj-shekhar1** [comment]
I am confirming the GV details, will update here.

**Comment by raj-shekhar1** [comment]
> I am confirming the GV and bootnode details, will update here.

LGTM, Thanks

**Review by raj-shekhar1 — APPROVED** [review]

**Review by lorenzo-dev1 — APPROVED** [review]
LGTM

File: params/config.go [hunk 2]
```diff
@@ -93,42 +92,76 @@ var (
...
-				BondedStake:   bigBondedStake,
-			}},
+			Validators: []*Validator{
+				{
+					// CTL GV-0
+					Treasury:      common.HexToAddress("0x75474aC55768fAb6fE092191eea8016b955072F5"),
+					OracleAddress: common.HexToAddress("0x6c5AE53a803796D788E917D1fE919BfC8B56d2E6"),
+					ConsensusKey:  common.Hex2Bytes("a3aa75e42e99275f7d7985538fedc06e7f128b138a5311702afc0dc129484763645c40c36fdd97ff0d0293b00a031714"),
+					Enode:         "enode://772248dfe1af5f77e0efc0510e83364bfad55cbd6d3e276f3bd0b4ddec6472aa98645655fd80bbf049ba3da18d219ab30a68fcb98da8e06dd42863dd0356cc95@35.242.168.170:30303",
+					BondedStake:   Ntn10000,
```
Comment (cmjc) [inline params/config.go:107]:
```suggestion
					BondedStake:   Ntn1,
```

Changing from `Ntn10000` to `NTN1` as requested by @spenot09

File: params/config.go [hunk 3]
```diff
@@ -93,42 +92,76 @@ var (
...
+				}, {
+					// CTL GV-1
+					Treasury:      common.HexToAddress("0x821BC352E77D885906B47001863f75e15C114f70"),
+					OracleAddress: common.HexToAddress("0x7C056299014D2F6f2e506ef1A4F89c94AAca004e"),
+					ConsensusKey:  common.Hex2Bytes("8a7474c5d53279bd21b8e0d0475ca6cd868155ac16d67d22c15eaee75f87101d9b329f4c2e2da52934a845fc4567254a"),
+					Enode:         "enode://22f696529d7874ca66d177c2c272600c3d1f2f7111d02140c462a8cbc789f5f8968c2ce57a5aac1373ef17bf3fc67d155b54691d1413516459824067e13750a4@34.92.27.46:30303",
+					BondedStake:   Ntn10000,
```
Comment (cmjc) [inline params/config.go:114]:
```suggestion
					BondedStake:   Ntn1,
```

Changing from `Ntn10000` to `NTN1` as requested by @spenot09

File: params/config.go [hunk 4]
```diff
@@ -93,42 +92,76 @@ var (
...
+				}, {
+					// CTL GV-2
+					Treasury:      common.HexToAddress("0x59e2EE43e1950a348ab3CC9b6Eb847C019c2CFb1"),
+					OracleAddress: common.HexToAddress("0x7615a195832843798DDb75B11CDA87D61C5F794D"),
+					ConsensusKey:  common.Hex2Bytes("866610ce1e9fc5871c7b99ee92e0aa3485c7b157bddda85518edf68aea68fe1ec1ccd7bb51064e3b742719c0cf562409"),
+					Enode:         "enode://24b2655b0434d1af4e2329cababf38963cab8a154e0b8c9748e75c85d10d7dab5032af7a41f3ec06dd1a7d3d306f1edee5dc46dad7a2858b80ebb56e5fa24925@34.233.111.193:30303",
+					BondedStake:   Ntn10000,
```
Comment (cmjc) [inline params/config.go:121]:
```suggestion
					BondedStake:   Ntn1,
```

Changing from `Ntn10000` to `NTN1` as requested by @spenot09

**Review by cmjc — CHANGES_REQUESTED** [review]
@yazzaoui - edits for the GV bondedStake amounts

**Pull request merged**